source/strings.tex

```diff
@@ -3617,7 +3617,7 @@
 \effects
 Equivalent to:
 \begin{codeblock}
-return basic_string_view<charT, traits>(this.data(), pos1, n1).compare(sv);
```

Comment:
The other `any::emplace`, for one. A quick search shows a few dozen `this->`s in the library clauses.